aws/lambda.tf

```diff
@@ -11,3 +11,16 @@ detect((when(errors > 10, lasting='5m'))).publish('AWS/Lambda function error rat
   }
 }
 
+resource "signalfx_detector" "lambda_historical_exectime_error" {
+    name = "[SFx] AWS/Lambda Historical Execution time error"
+    description = "AWS/Lambda Lambda execution time has been greater then historical norm during the past 10 minutes"
+    program_text = <<-EOF
+from signalfx.detectors.against_periods import against_periods
+A = data('aws.lambda.executionTime').publish(label='A', enable=False)
```

Comment:
aws.lambda.executionTime - where does this metric come from? It not a Cloudwatch or Wrapper one from what I can tell.